### ISSUE TITLE
Consider hook area constants

### ISSUE THREAD

**Issue by Guite** [Guite/MostGenerator#739]
It could make sense to introduce constants for hook area names, similar to how @craigh did it in the Users module here: https://github.com/zikula/core/compare/75f84da830...92df02618a

Should be reviewed after #260 has been done.

**Comment by Guite** [comment]
Closing as "won't fix", as this would make things more cumbersome in the `HookHelper` class.
